docs/resources.md

```diff
@@ -356,6 +356,7 @@ The Cluster resource has the following parameters:
 -   `url` (required): Host url of the master node
 -   `username` (required): the user with access to the cluster
 -   `password`: to be used for clusters with basic auth
+-   `namespace`: The namespace to be given to the target namespace
```

Comment:
Suggest changing this to `The namespace to target in the cluster`

Reply:
Thanks for your suggestion!